doc/user/content/sql/create-source/text-s3.md

```diff
@@ -102,6 +102,19 @@ for all the lines that have `/updates` as their exact path:
 12/31/2020:23:55:59 | 99.99.44.44    | Python/Requests 22
 ```
 
+### Example Single Object
```

Comment:
Nit: docs use sentence case for headings, so "Example single object". I think just leave it for now because this entire doc is inconsistent about it, and I'll submit a separate PR to clean it up.